src/trixi.c

```diff
@@ -0,0 +1,83 @@
+#include <stdio.h>
+#include <julia.h>
+#include <mpi.h>
+
+#include "trixi.h"
+
+
+
+#define LOC __func__, __FILE__, __LINE__
+
+// Auxiliary function to evaluate julia REPL string with exception handling
+static jl_value_t* checked_eval_string(const char* code, const char* func, const char* file, int lineno);
+
+
+
+void trixi_initialize(MPI_Fint* comm) {
+
+    // Convert Fortran communicator type to C
+    MPI_Comm comm_c = MPI_Comm_f2c(*comm);
+
+    // Init Julia
+    jl_init();
+
+    // @todo
+    // Activate julia environment
+    // checked_eval_string("using Pkg; Pkg.activate(\".\"); Pkg.status()", LOC);
+
+    // Test Julia
+    checked_eval_string("println(\"libtrixi: Hi, I'm julia!\")", LOC);
+
+    // @todo
+    // Load Trixi
+    // checked_eval_string("using Trixi; trixi_include(default_example())", LOC);
+}
+
+
+void trixi_finalize() {
+
+    jl_atexit_hook(0);
+}
+
+
+double trixi_get_timestep() {
+
+    return 1.0;
+}
+
+
+void trixi_integrate() {
+
+}
+
+
+void julia_eval_repl(const char * code) {
```

Comment:
```suggestion
void julia_eval_string(const char * code) {
```
I wouldn’t call it REPL since technically it is not a REPL, and it behaves a little different.